pymc/logprob/transforms.py

```diff
@@ -1229,6 +1232,10 @@ def transformed_logprob(op, values, *inputs, use_jacobian=True, **kwargs):
                 assert isinstance(value.owner.op, TransformedVariable)
                 original_forward_value = value.owner.inputs[1]
                 jacobian = transform.log_jac_det(original_forward_value, *inputs).copy()
+                # Check if jacobian dimensions align with logp
+                if jacobian.ndim < logp.ndim:
+                    diff_ndims = logp.ndim - jacobian.ndim
+                    logp = logp.sum(axis=np.arange(-diff_ndims, 0))
```

Comment:
This line only happens if `use_jacobian` is `True`. If it's false, but the transform is still applied, should you still `sum` across the extra dimensions in the `logp`? If you don't then you'll be returning an inconsistently shaped tensor depending on this flag.

Reply:
Yeah, let's do it. The jacobian flag is a bit hacking, but I think it's better to return the same shapes.